extensions/graphite-emitter/README.md

```diff
@@ -0,0 +1,50 @@
+## introduction
+
+This extension emits some selected druid metrics to a graphite carbon server.
+Events are sent after been [pickled](http://graphite.readthedocs.org/en/latest/feeding-carbon.html#the-pickle-protocol) , the size of the batch is configurable.
+Since Graphite doesn't support alerting, alerts will be redirect to warning logs by default. 
+
+ Metrics are organized using the following schema:
+`<namespacePrefix>.[<druid service name>].[<druid hostname>].<druid metrics dimensions>.<druid metrics name>`
+Properly naming your metrics is critical to avoid conflicts, confusing data and potentially wrong interpretation later on.
+
+Example `druid.historical.hist-host1.yahoo.com:8080.MyDataSourceName.GroupBy.query/time`:
+
+ * `druid` -> namespace prefix 
+ * `historical` -> service name 
+ * `hist-host1.yahoo.com:8080` -> druid hostname
+ * `MyDataSourceName` -> dimension value 
+ * `GroupBy` -> dimension value
+ * `query/time` -> metric name
+
+User has control of `<namespace>.[<druid service name>].[<druid hostname>].`
```

Comment:
I see, thanks.  The flow of this text led me to think it would be discussed in the following paragraph along with control of the host and service name.  It would remove this question from a reader's mind if you add one sentence to the following paragraph, saying "You can set the namespace with the value of druid.emitter.graphite.namespacePrefix.  It would be consistent to mention all 3 in the paragraph, or leave all 3 for the configuration table.

Having said all that, if you want to just leave this alone, I'm ok with that.